scripts/analytics/file_summary.py

```diff
@@ -0,0 +1,78 @@
+import django
+django.setup()
+
+from django.db.models import Q
+import pytz
+import logging
+from dateutil.parser import parse
+from datetime import datetime, timedelta
+from django.utils import timezone
+
+from website.app import init_app
+from scripts.analytics.base import SummaryAnalytics
+
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(level=logging.INFO)
+
+
+class FileSummary(SummaryAnalytics):
+
+    @property
+    def collection_name(self):
+        return 'file_summary'
+
+    def get_events(self, date):
+        super(FileSummary, self).get_events(date)
+        from addons.osfstorage.models import OsfStorageFile
+
+        # Convert to a datetime at midnight for queries and the timestamp
+        timestamp_datetime = datetime(date.year, date.month, date.day).replace(tzinfo=pytz.UTC)
```

Comment:
Personal preference: no need to import pytz and use pytz.UTC. Instead you can just use timezone.utc from django.utils